Tests/SWBBuildSystemTests/DependencyValidationTests.swift

```diff
@@ -493,7 +494,7 @@ fileprivate struct DependencyValidationTests: CoreBasedTests {
                             TestStandardTarget(
                                 "CoreFoo", type: .framework,
                                 buildPhases: [
-                                    TestSourcesBuildPhase(["CoreFoo.m"]),
+                                    TestSourcesBuildPhase(["CoreFoo.m", "CoreBar.m"]),
```

Comment:
This makes it so that the test would see duplicate diagnostics in the old implementation which this PR then fixes.